server/routes/image.js

```diff
@@ -87,4 +89,30 @@ router.post('/student/:studentId', async (req, res, next) => {
     }
 });
 
+// TODO: this potentially leaves S3 bucket being polluted with images, but these images will never be associated to any user
+router.post('/add', async (req, res, next) => {
+    try {
+        const form = new multiparty.Form();
+        form.parse(req, async (error, fields, files) => {
+            if (!error) {
+                const email = fields.email[0]
+                const emailHash = await bcrypt.hash(email, HASH_COST);
+                const path = files.imageFile[0].path;
+                const buffer = fs.readFileSync(path);
+                const type = await fileType.fromBuffer(buffer);
+                const fileName = `${emailHash}-${moment().format('MM-DD-YYYY')}`;
+                const data = await uploadFile(buffer, fileName, type);
+                let imageLocation = data.Location;
+                res.status(200).send({
+                    success: true,
+                    imageUrl: imageLocation
+                })
+            }
+        })
+    } catch (e) {
+        console.log("Error: image#addImage", e);
+        res.status(500).send({'error' : e});
+    }
+});
+
```

Comment:
I saw your text earlier - this solution is fine for now, especially since it works and has the behavior we want
One option we have (That can be filed under Nice to Have alongside the mongo expiry) is having a temporary bucket for images that then moves images to the standard bucket upon verification and deletes old files automatically (https://stackoverflow.com/questions/46769995/temporarily-upload-on-s3-and-remove-if-its-not-used)